staramr/subcommand/Search.py

```diff
@@ -129,17 +137,18 @@ def _setup_args(self, arg_parser):
 
         return arg_parser
 
-    def _print_dataframes_to_excel(self, outfile_path, summary_dataframe, resfinder_dataframe, pointfinder_dataframe,
+    def _print_dataframes_to_excel(self, outfile_path, summary_dataframe, resfinder_dataframe, pointfinder_dataframe, plasmidfinder_dataframe, 
                                    settings_dataframe):
         writer = pd.ExcelWriter(outfile_path, engine='xlsxwriter')
 
         sheetname_dataframe = {}
         sheetname_dataframe['Summary'] = summary_dataframe
         sheetname_dataframe['ResFinder'] = resfinder_dataframe
+        sheetname_dataframe['PlasmidFinder'] = plasmidfinder_dataframe
         if pointfinder_dataframe is not None:
             sheetname_dataframe['PointFinder'] = pointfinder_dataframe
 
-        for name in ['Summary', 'ResFinder', 'PointFinder']:
+        for name in ['Summary', 'ResFinder', 'PointFinder', 'PlasmidFinder']:
```

Comment:
I :heart: that you've taken the time to get **PlasmidFinder** to write to the excel file :smile:.

Reply:
Thanks! :smile: